build/package/generate-msi.sh

```diff
@@ -51,6 +51,12 @@ go-msi make --path "${WIX_MANIFEST_FILE}"  --msi "dist/${PACKAGE_NAME}" --versio
 
 go run ./tools/sign -file "dist/${PACKAGE_NAME}"
 
+CHOCOLATEY_PACKAGE_PATH=build/package/chocolatey/temp
+CHOCOLATEY_PACKAGE_NAME="atlascli.${VERSION}.nupkg"
```

Comment:
Why keep them separated? Why not just one file path?

Reply:
Because I'm changing path where we're executing commands in `cmd.Dir = path` and just telling choco to push package `name`. 
```
func update(path, name, secret string) error {
	cmd := exec.Command("choco", "push", name, "--api-key", secret)
	cmd.Dir = path
	err := cmd.Start()
	return err
}
```

Reply:
I don't think you need to set `Dir` if you send the path altogether

Reply:
Corrected